observatorium/base/observatorium.yaml

```diff
@@ -0,0 +1,97 @@
+---
+apiVersion: core.observatorium.io/v1alpha1
+kind: Observatorium
+metadata:
+  labels:
+    app.kubernetes.io/instance: opf-observatorium
+    app.kubernetes.io/name: observatorium-cr
+    app.kubernetes.io/part-of: observatorium
```

Comment:
What about moving (some of) this to `commonLabels` in `kustomization.yaml`, so it gets applied to the rest of the resources as well.